integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/TraefikParams.java

```diff
@@ -72,6 +73,7 @@ public Map<String, Object> getValues() {
     }
 
     values.put(TRAEFIK_IMAGE, traefikImage);
+    values.put(TRAEFIK_IMAGE_REGISTRY, "");
```

Comment:
I think you should change this to 
values.put(TRAEFIK_IMAGE_REGISTRY, "phx.ocir.io")
and update TRAEFIK_INGRESS_IMAGE_NAME to:
"weblogick8s/test-images/traefik-ingress/traefik"

Reply:
yes changed it